numba_dpex/tests/experimental/kernel_api_overloads/spv_overloads/test_atomic_ref.py

```diff
@@ -54,13 +59,48 @@ def test_atomic_ref_compilation_failure():
     """
 
     @dpex_exp.kernel
-    def atomic_ref_kernel(item: Item, a, b):
+    def atomic_ref_kernel(item: kapi.Item, a, b):
         i = item.get_id(0)
-        v = AtomicRef(b, index=0, address_space=AddressSpace.LOCAL)
+        v = kapi.AtomicRef(b, index=0, address_space=kapi.AddressSpace.LOCAL)
         v.fetch_add(a[i])
 
     a = dpnp.ones(10)
     b = dpnp.zeros(10)
 
     with pytest.raises(TypingError):
-        dpex_exp.call_kernel(atomic_ref_kernel, Range(10), a, b)
+        dpex_exp.call_kernel(atomic_ref_kernel, kapi.Range(10), a, b)
+
+
+def test_atomic_ref_compilation_local_accessor():
+
```

Comment:
A little bit of NITpicking, but let's be consistent, having leading empty line in function body never helps. We should set linter to check it automatically.

Reply:
Added docstring. But cannot remove the newline somehow, VSCode's autoformat setting in my system adds it.